homeassistant/components/suez_water/sensor.py

```diff
@@ -76,7 +78,9 @@ def _fetch_data(self):
         try:
             self.client.update()
             # _state holds the volume of consumed water during previous day
-            self._attr_native_value = self.client.state
+            if self.client.state >= 0:
```

Comment:
If `self.client.state` can be `None` we need to handle that first. We can't compare `None` with greater than.

```
Python 3.11.6 (main, Oct 23 2023, 22:47:21) [GCC 9.4.0]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.14.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: None >= 0
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[1], line 1
----> 1 None >= 0

TypeError: '>=' not supported between instances of 'NoneType' and 'int'
```